src/services/mhs/logging-utils.js

```diff
@@ -0,0 +1,88 @@
+import { logError, logInfo } from '../../middleware/logging';
+
+export const LOG_SIZE_LIMIT = 256 * 1024; // the 256 KB size limit of Cloud Watch
+
+export const logOutboundMessage = message => {
+  /**
+   * @param {object} message
+   * Helper function to log an outbound message within the 256KB size limit of Cloud Watch.
+   * It will attempt to do this by two steps:
+   * 1. Remove any base64 content in the message
+   * 2. If the message is still > 256KB without the base64 content, split the message into multiple lines
+   *
+   * Internally it is just composing removeBase64Payloads -> checkSizeAndLogMessage.
+   */
+  if (typeof message !== 'object')
+    logInfo('logOutboundMessage got an input which is not a JS object. May not work as expected.');
+
+  try {
+    checkSizeAndLogMessage(removeBase64Payloads(message));
+  } catch (error) {
+    // catch any error and let the original workflow to continue
+    logError('Encountered error while trying to log the outbound message', error);
+  }
+};
+
+export const removeBase64Payloads = inputObject => {
```

Comment:
Could `inputObject` be renamed to `message`?

```suggestion
export const removeBase64Payloads = message => {
```

Reply:
Thanks! For this one actually I thought the same at first, but later changed my mind not to use the name 'message' for argument here. 
Because this function is recursive, so not only the message itself but also every children of the message object will be passed as the argument at some point. So I didn't call the input param 'message'.